crates/ruff_dev/src/generate_ty_rules.rs

```diff
@@ -56,6 +56,10 @@ fn generate_markdown() -> String {
 
     let mut output = String::new();
 
+    let _ = writeln!(
+        &mut output,
+        "<!-- WARNING: This file is auto-generated. Edit 'generate_ty_rules.rs' instead. -->\n"
```

Comment:
Same here: Users shouldn't need to change the script. Instead, they should update the metadata of the corresponding `declare_lint!`